app/containers/Zapper/saga.js

```diff
@@ -284,8 +291,15 @@ function* zapOut(action) {
     yield call(web3.eth.sendTransaction, zapOutTransaction);
   } catch (error) {
     console.log('Zap Failed', error);
+    let errorMessage = '';
+    if (error && error.message) {
+      errorMessage = error.message;
+    } else {
+      errorMessage =
+        'This means there is not enough liquidity for this token or Zapper may be down. Try with DAI, ETH, USDC or USDT as they offer the more liquidity. ';
```

Comment:
@dudesahn what do you think about this text? Do you think we should use something more generic or does this work?

Reply:
Do we also display error messages for the other errors? Would be good to do for all of them, and style them in the same way @patcito. I'm pretty sure this is the generic/black-box error, but I know zapper has more specific ones as well.

Reply:
They return a generic error that says "low level erc20 transaction failed" most of the time.